server.go

```diff
@@ -78,7 +96,7 @@ func main() {
 		panic(err)
 	}
 
-	db, _ := sql.Open("mysql", fmt.Sprintf("%s:%s@/", *mysqlUser, *mysqlPassword))
+	db, _ := sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s:%d)/", *mysqlUser, *mysqlPassword, *mysqlHost, *mysqlPort))
```

Comment:
Good point. I think this is outside of the scope of this pull request however. Opened issue https://github.com/cloudfoundry-incubator/galera-healthcheck/issues/3